plugins/traefik/main_test.go

```diff
@@ -61,6 +67,90 @@ func TestSablierMiddleware_ServeHTTP(t *testing.T) {
 				},
 			},
 			expected: "response from sablier",
+			code:     200,
+		},
+		{
+			name: "sablier service is ready but 503",
+			sablier: sablier{
+				headers: map[string]string{
+					"X-Sablier-Session-Status": "ready",
+				},
+				body: "response from sablier",
+			},
+			fields: fields{
+				Next: http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+					w.WriteHeader(http.StatusServiceUnavailable)
+				}),
+				Config: &Config{
+					SessionDuration: "1m",
+					Dynamic:         &DynamicConfiguration{},
+				},
+			},
+			expected: "response from sablier",
+			code:     200,
+		},
+		{
+			name: "sablier service is ready blocking",
+			sablier: sablier{
+				headers: map[string]string{
+					"X-Sablier-Session-Status": "ready",
+				},
+				body: "response from sablier",
+			},
+			fields: fields{
+				Next: http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+					httptrace.ContextClientTrace(r.Context()).WroteHeaders()
+					fmt.Fprint(w, "response from service")
+				}),
+				Config: &Config{
+					SessionDuration: "1m",
+					Blocking:        &BlockingConfiguration{},
+				},
+			},
+			expected: "response from service",
+			code:     200,
+		},
+		{
+			name: "sablier service is not ready blocking",
+			sablier: sablier{
+				headers: map[string]string{
+					"X-Sablier-Session-Status": "not-ready",
+				},
+				body: "response from sablier",
+			},
+			fields: fields{
+				Next: http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+					httptrace.ContextClientTrace(r.Context()).WroteHeaders()
+					fmt.Fprint(w, "response from service")
+				}),
+				Config: &Config{
+					SessionDuration: "1m",
+					Blocking:        &BlockingConfiguration{},
+				},
+			},
+			expected: "response from sablier",
+			// is this correct for blocking? I would expect to get error
```

Comment:
This seems to be wired to return 200 and json but this is how it was working before as well.